flume-ng-core/src/main/java/org/apache/flume/source/StressSource.java

```diff
@@ -89,6 +91,13 @@ protected void doConfigure(Context context) throws FlumeException {
     /* Size of events to be generated. */
     int size = context.getInteger("size", 500);
 
+    int rateLimit = context.getInteger("maxEventsPerSecond", 0);
```

Comment:
Could you please update the documentation with the new parameter?

Reply:
Absolutely, thought I had!!